src/components/LastDeliveredCell.tsx

```diff
@@ -0,0 +1,30 @@
+import { DefaultServerCellComponentProps } from 'payload'
+import { getTimeAgo } from '@/lib/utils'
+
+const LastDeliveredCell = async ({ rowData, payload, collectionSlug }: DefaultServerCellComponentProps) => {
+  const customerId = collectionSlug === 'customers' ? rowData.id : rowData.customer; 
+  const transactionAt = (await payload.find({
+    collection: 'transaction',
+    where: {
+      customer: {
+        equals: customerId,
+      },
+      bottleGiven: {
+        greater_than: 0,
+      }
+    },
+    limit: 1,
+    sort: '-transactionAt',
+    select: {
+      transactionAt: true,
+    }
+  })).docs[0]?.transactionAt;
+
+  return (
+    <div>
+      {getTimeAgo(transactionAt)}
+    </div>
+  )
```

Comment:
@shaheem-khanzada we do not need `getTimeAgo()` you can remove that.
```suggestion
  if (!transactionAt) return 'No delivery yet'
  return (
    <div>
      {formatDistance(transactionAt, new Date(), { addSuffix: true })}
    </div>
  )
```